clang/lib/Format/Format.cpp

```diff
@@ -807,12 +807,18 @@ template <> struct MappingTraits<FormatStyle> {
         FormatStyle PredefinedStyle;
         if (getPredefinedStyle(StyleName, Style.Language, &PredefinedStyle) &&
             Style == PredefinedStyle) {
-          IO.mapOptional("# BasedOnStyle", StyleName);
```

Comment:
is the a way to emit the a comment into the YAML at the correct indentation level?

Reply:
I don't think our YAML writer supports comments at all.

Reply:
How hard would it be to add that?

Reply:
I don't know, but I'd prefer to avoid supporting it if it isn't even needed. We can still add it if we want this back.